packages/plugin-core/package.json

```diff
@@ -771,6 +777,12 @@
         "key": "ctrl+l",
         "when": "dendron:pluginActive"
       },
+      {
+        "command": "dendron.lookupNoteAutoComplete",
+        "mac": "Tab",
```

Comment:
`key` alone should be good enough, `mac` is only if we have mac specific keybindings

Reply:
removed